samples/algorithms/Entanglement.qs

```diff
@@ -0,0 +1,32 @@
+/// # Sample
+/// Entanglement
```

Comment:
I find these samples so nice and simple - I think we should include them with the other samples in our playground (in samples.mjs).
Superposition should probably be included before the BellState, and RandomBit before the Random Number Generator. In that case BellState may have tuples.
Entanglement, however, seems to be exactly the same as the BellState. Do we need it at all?

Reply:
I'll add them to the playground in the order that you suggested :)

They are the same. I could expand the Bell state sample to cover all four bell states.

Reply:
Yes, this is interesting. We may add Entanglement after Superposition, but before the BellState. In this case we can indeed show all four Bell states and we don't need as many comments. Great idea!
The only question is how to do this so that the histogram shows the results nicely. Maybe let user uncomment one of the four preparation routines?

Reply:
We may have something like this:
```qsharp
    @EntryPoint()
    operation Main() : (Result, Result) {
        // Allocate the two qubits that will be used to create a Bell state.
        use q1 = Qubit();
        use q2 = Qubit();

        // Uncomment any or both of the X gates to obtain different Bell states.
        // X(q1);
        // X(q2);
    
        BellState(q1, q2)
    }
```

Reply:
I ended up returning measurement results for all Bell states and doing a per-state preparation operation.